test/distributed/_composable/test_fully_shard.py

```diff
@@ -589,12 +599,57 @@ def _check_model_parity(self, m1: nn.Module, m2: nn.Module):
         """
         Checks that m1 and m2 have equivalent named_parameters.
         """
-        for (n1, p1), (n2, p2) in zip(
-            m1.named_parameters(), m2.named_parameters()
-        ):
+        for (n1, p1), (n2, p2) in zip(m1.named_parameters(), m2.named_parameters()):
             self.assertEqual(n1, n2)
             self.assertEqual(p1, p2)
 
 
+class TestFSDPOptimStateDict(FSDPTest):
+    """Composable FSDP optimizer state dict tests."""
+
+    @property
+    def world_size(self) -> int:
+        return 2
+
+    @skip_if_lt_x_gpu(2)
+    def _test_optim_state_dict_save_load(self):
```

Comment:
Might be better to just have `test` instead of disabling with prefix, and adding skip decorator mentioning reason it is disabled and